app/src/main/java/ui/HomeViewModel.java

```diff
@@ -0,0 +1,13 @@
+package ui;
+
+import android.app.Application;
+
+import androidx.annotation.NonNull;
+import androidx.lifecycle.AndroidViewModel;
+
+public class HomeViewModel extends AndroidViewModel {
+
+    public HomeViewModel(@NonNull Application application) {
+        super(application);
+    }
+}
```

Comment:
This is a very picky style suggestion but you should add a newline to the end of every file. https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file